tests/unit_tests/test_mean.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def mean_process_code():
+    return load_process_code("mean")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [([1, 3, 5, 7], 4), ([6, 6], 6), ([1, 0, 3, 2], 1.5)],
```

Comment:
They were intentionally omitted (they are taken from https://docs.openeo.cloud/processes/#mean) because the process doesn't work for them and the tests would fail.

After this PR is merged I'll write issues for writing tests for all existing processes, including this one. I just didn't want to change the process definition in this PR.